lib/generators/rolemodel/source_map/source_map/config/initializers/source_map.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+Rails.application.configure do
+  #RoleModel::SourceMap(allowed_users_emails[, **options])
+  #allowed_users_emails should be a String or a Array of Strings
```

Comment:
Grammatical error: Should be `an Array` instead of `a Array`

Reply:
Resolved.